spec/spec_helper.cr

```diff
@@ -43,6 +41,12 @@ Fixtures.reset_config
 # Run the server
 DppmRestApi.run Socket::IPAddress::LOOPBACK, DPPM::Prefix.default_dppm_config.port, __DIR__
 # Set all configs back to the expected values, in case they changed
-Spec.before_each { Fixtures.reset_config }
+Spec.before_each do
+  Fixtures.reset_config
+  FileUtils.mkdir_p Fixtures::PREFIX_PATH
```

Comment:
Is there a way for DPPM to instantiate the directory structure that it needs? I'm getting a *no such file or directory* error when trying to clean packages on an empty prefix.

Reply:
Yes, `Prefix#create`